docs/_getting-started-windows-android.md

```diff
@@ -57,10 +57,10 @@ To do that, open Android Studio, click on "More Actions" button and select "SDK
 
 Select the "SDK Platforms" tab from within the SDK Manager, then check the box next to "Show Package Details" in the bottom right corner. Look for and expand the `Android 12 (S)` entry, then make sure the following items are checked:
```

Comment:
`Android 12 (S)` here is wrong. It should be `Android 13 (Tiramisu)`.
This is also wrong in every other page (win/mac/linux). Are you up for fixing it @arushikesarwani94 ?

Reply:
Ahh missed that. Good catch. Will fix it.